packages/@react-stately/collections/src/useCollectionState.ts

```diff
@@ -48,7 +49,7 @@ export function useCollectionState<T extends object, V, W>(opts: CollectionProps
   let [contentSize, setContentSize] = useState(new Size());
   let [isAnimating, setAnimating] = useState(false);
   let [isScrolling, setScrolling] = useState(false);
-  let collectionManager = useMemo(() => new CollectionManager<T, V, W>(), []);
+  let collectionManager = useMemo(() => new CollectionManager<T, V, W>({transitionDuration: opts.transitionDuration}), []);
```

Comment:
I think we can reuse the existing collection manager when the transition duration changes. Maybe set it below like the layout/collection props?